lib/build-scss.js

```diff
@@ -94,7 +94,7 @@ const compileAndWriteStyleSheets = ({
       },
     }],
     // For now we can't resolve these warnings as we need to upgrade our 'bootstrap' dependency to do so:
-    silenceDeprecations: ['abs-percent', 'color-functions', 'import', 'mixed-decls', 'global-builtin'],
+    silenceDeprecations: ['abs-percent', 'color-functions', 'import', 'mixed-decls', 'global-builtin', 'legacy-js-api'],
```

Comment:
Do we have a general issue tracking deprecated things we can't avoid using because of bootstrap? With a quick search I found https://github.com/openedx/paragon/issues/3425 but that doesn't quite fit.

Reply:
I don't believe so, not that I'm aware of or can find in searches. These fixes have been fairly adhoc by various folks.